test/e2e/specs/editor/blocks/navigation.spec.js

```diff
@@ -993,11 +993,9 @@ test.describe( 'Navigation block - Frontend interactivity', () => {
 			await expect( overlayMenuFirstElement ).toBeFocused();
 
 			// Test: overlay menu traps focus
-			await page.keyboard.press( 'Tab' );
-			await page.keyboard.press( 'Tab' );
+			await pageUtils.pressKeys( 'Tab', { times: 2, delay: 50 } );
```

Comment:
I love that it solves the issue and I have no problem merging this. I'm just wondering if there's an actual bug that makes us use `delay`, which is a last resort? It feels weird to me that Playwright has no problem tabbing through elements in the editor but is "too fast" in the frontend 🤔 . Maybe there's something obvious I'm missing though!